cg/constants/constants.py

```diff
@@ -52,6 +52,12 @@
 STATUS_OPTIONS = ("affected", "unaffected", "unknown")
 
 
+class GenomeVersions(StrEnum):
```

Comment:
```suggestion
class GenomeVersion(StrEnum):
```
It seems to be most common to name Enums as singular, see https://stackoverflow.com/questions/1335959/singular-or-plural-for-enumerations

Reply:
Good call 🏅

Reply:
Thanks for reviewing it Patrik!